src/libponyc/pass/docgen.c

```diff
@@ -344,7 +344,7 @@ static const char* doc_get_cap(ast_t* cap)
 
 // Write the given type to the current type file
 static void doc_type(docgen_t* docgen, docgen_opt_t* docgen_opt,
-  ast_t* type, bool generate_links)
+  ast_t* type, bool generate_links, bool break_union_lines)
```

Comment:
maybe we want to rename this to `break_linkes` as it does not only apply to union types and would be more consistent with the existing functions.

Reply:
hmm yeah... it's a bit odd there. I initially only wanted it to apply to union, but then I've found that's getting threaded through the other types... so, yeah, where this ended up in, it makes more sense to call it `break_lines`.